.github/workflows/codeql.yml

```diff
@@ -0,0 +1,72 @@
+# For most projects, this workflow file will not need changing; you simply need
+# to commit it to your repository.
+#
+# You may wish to alter this file to override the set of languages analyzed,
+# or to provide custom queries or build logic.
+#
+# ******** NOTE ********
+# We have attempted to detect the languages in your repository. Please check
+# the `language` matrix defined below to confirm you have the correct set of
+# supported CodeQL languages.
+#
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ "develop", hotfix/*, master, release/*, feature/** ]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [ "develop", hotfix/*, master, release/* ]
+
+jobs:
+  analyze:
+    name: Analyze
+    runs-on: ubuntu-latest
+    permissions:
+      actions: read
+      contents: read
+      security-events: write
+
+    strategy:
+      fail-fast: false
+      matrix:
+        language: [ 'javascript' ]
```

Comment:
The link you have on line 35 says they support typescript. Should this be typescript?

Reply:
Interesting, the check actually seems to notice and automatically switches anyway (maybe its a common mistake). 
Switching anyway